app/src/common/utils/fetch.js

```diff
@@ -48,7 +48,7 @@ export const initAuthInterceptor = (store) => {
     (error) => {
       if (error.response && error.response.status === 401) {
         const isAuthorized = isAuthorizedSelector(store.getState());
-        isAuthorized && store.dispatch(logoutAction());
+        !isAuthorized && store.dispatch(logoutAction());
```

Comment:
This fix was created to avoid unnecessary dispatch of  `logoutAction` for unauthorized user. It means that if the user is not already in the system, this action shouldn't be dispatched.

Reply:
Fixed.